CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Added Features
 - ROX-18689: ACS will qualify the registry (and path) of images from the container runtime when env var `ROX_UNQUALIFIED_SEARCH_REGISTRIES` is set to `true` on both Central and Sensor.
   - This enables support for CRI-O's unqualified search registries and short name aliases ([more info](https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md)).
+  - ROX-24725: Enhances Sensor's image scan event handling when `ROX_UNQUALIFIED_SEARCH_REGISTRIES` is `true` so that only one simultaneous scan request is allowed per unique image, and increases the chances of scan cache hits when multiple names for the same image have been observed.  This enhancement can be disabled by setting the Sensor env var `ROX_SENSOR_SINGLE_SCAN` to `false`.
```

Comment:
I think there is a spacing problem for the bullet point?

nit: `so that only one` -> can remove `that`

nit: `, and increases the chances of scan cache hits when multiple names for the same image have been observed.` -> can make this a sub bullet point so we don't have one long sentence

`This enhancement can be disabled by setting the Sensor env var `ROX_SENSOR_SINGLE_SCAN` to `false`.` -> I think we should also make it clear this currently only works when it's enabled (maybe enough to say this defaults to `true`)

Reply:
> I think there is a spacing problem for the bullet point?

That was intentional to associate the update with unqualified search registries feature.  I moved the whole thing to the `Technical Changes` section instead.